lib/rules/web/admin_console/4.16/logs.xyaml

```diff
@@ -99,11 +99,11 @@ check_event_is_streaming:
 check_logs_not_wrapped:
   elements:
   - selector:
-      xpath: //div[contains(@class,'pf-c-log-viewer__scroll-container') and contains(@style,'auto')]
+      xpath: //div[contains(@class,'pf-v5-c-log-viewer__scroll-container') and contains(@style,'auto')]
 check_logs_wrapped:
   elements:
   - selector:
-      xpath: //div[contains(@class,'pf-c-log-viewer__scroll-container') and contains(@style,'hidden')]
+      xpath: //div[contains(@class,'pf-v5-c-log-viewer__scroll-container') and contains(@style,'hidden')]
```

Comment:
About all pf5 related rules here, suggest update like: //div[contains(@class,'log-viewer__scroll-container') and contains(@style,'hidden')]